doc/source/tune/tutorials/tune-storage.rst

```diff
@@ -3,8 +3,8 @@
 How to Configure Storage Options for a Distributed Tune Experiment?
 ===================================================================
 
-When running Tune in a distributed setting, trials run on many different machines,
-which means that experiment outputs such as model checkpoints will be spread all across the cluster.
+Before diving into storage options, one can take a look at
+:ref:`the different types of data stored by Tune <tune-persisted-experiment-data>`.
```

Comment:
Let's also make this a `seealso` after the intro text, right before `Storage options with tune`